Digital_Feed_7e2/Digital_Feed_7e2.ino

```diff
@@ -134,16 +134,16 @@ char LCD_Row_2[17];
 
 
 // ***** Encoder *****
-#define ENC_TICK              (ENC_LINE_PER_REV * 2)    // Рабочее кол-во импульсов
+#define ENC_TICK              (ENC_LINE_PER_REV * 2)    // Working number of impulses
 #define Encoder_Init()         DDRD = B00000000;\
-                               PORTD = B11111111        // подтяжка PIN_21, 20, 19, 18
+                               PORTD = B11111111        // tightening PIN_21, 20, 19, 18
```

Comment:
use either "pull up" or "pull down" depending on whether pins are being pulled to 5V or 0